packages/connector-aliyun-sms/package.json

```diff
@@ -0,0 +1,57 @@
+{
+    "name": "@logto/connector-aliyun-sms",
+    "version": "0.1.0",
+    "description": "Aliyun SMS connector implementation.",
+    "main": "./lib/index.js",
+    "exports": "./lib/index.js",
+    "author": "Silverhand Inc. <contact@silverhand.io>",
+    "license": "MPL-2.0",
+    "files": [
+        "lib",
+        "docs"
+    ],
+    "private": false,
+    "scripts": {
+        "preinstall": "npx only-allow pnpm",
+        "precommit": "lint-staged",
+        "build": "rm -rf lib/ && tsc -p tsconfig.build.json",
+        "lint": "eslint --ext .ts src",
+        "lint:report": "pnpm lint -- --format json --output-file report.json",
+        "dev": "rm -rf lib/ && tsc-watch -p tsconfig.build.json --preserveWatchOutput --onSuccess \"node ./lib/index.js\"",
+        "test": "jest",
+        "test:coverage": "jest --coverage --silent",
+        "prepack": "pnpm build"
+    },
+    "dependencies": {
+        "@logto/connector-types": "^0.1.0",
+        "@logto/jest-config": "^0.1.0",
+        "@logto/shared": "^0.1.0",
+        "@logto/schemas": "^0.1.0",
+        "@silverhand/essentials": "^1.1.0",
+        "got": "^11.8.2",
+        "nanoid": "^3.1.23",
```

Comment:
Searched the current master branch and found that we used `nanoid` in tests in the past. 
![image](https://user-images.githubusercontent.com/10594507/165443987-9fc5378c-7863-4afd-9d30-120a479c1fde.png)

since no longer use `nanoid` in this PR now (`connector-aliyun-sms`),
we should remove it here.

Reply:
removed, thanks!